analysis/question_4/suicide_rate_distribution_analysis_byage.py

```diff
@@ -0,0 +1,31 @@
+# This code is the analysis on the top 5 countries with hight suicide rates
+# It further attempts to explore their differences and similarity of suicide rates in different age groups
+# barplot from Seaborn library is used for visualisation 
+
+import os
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+
+# Getting Data of crude suicide rate
+suicide_rate_data_filepath = os.path.join("..","processed_data", "crude_suicide_rates.csv")
+suicide_rate_dataframe = pd.read_csv(suicide_rate_data_filepath, index_col=0)
```

Comment:
- how if line 12 and 13 merged into one line
`suicide_rate_dataframe = pd.read_csv('../processed_data/crude_suicide_rates.csv')`

-  do index_col really needed here?